demo1.py

```diff
@@ -0,0 +1,5 @@
+def func(is_true):
+    if is_true:

```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)